test/metabase/actions_test.clj

```diff
@@ -2,14 +2,22 @@
   (:require
    [clojure.test :refer :all]
    [metabase.actions :as actions]
+   [metabase.actions.execution :as actions.execution]
    [metabase.api.common :refer [*current-user-permissions-set*]]
    [metabase.driver :as driver]
    [metabase.models :refer [Database Table]]
+   [metabase.models.action :as action]
    [metabase.query-processor :as qp]
    [metabase.test :as mt]
    [metabase.util :as u]
    [metabase.util.schema :as su]
-   [schema.core :as s]))
+   [schema.core :as s]
+   [toucan.db :as db])
```

Comment:
I've been trying to write new code with `t2` instead of `db`

Reply:
good call. Switching